tests_async/transport/test_aiohttp_req.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import aiohttp
+from aioresponses import aioresponses
+import mock
+import pytest
+from tests_async.transport import async_compliance
+
+import google.auth.credentials_async
+from google.auth.transport import aiohttp_req
+import google.auth.transport._mtls_helper
+
+
+class TestRequestResponse(async_compliance.RequestResponseTests):
+    @pytest.mark.asyncio
+    def make_request(self):
+        return aiohttp_req.Request()
+
+    def test_timeout(self):
+        http = mock.create_autospec(aiohttp.ClientSession, instance=True)
+        request = google.auth.transport.aiohttp_req.Request(http)
+        request(url="http://example.com", method="GET", timeout=5)
+
```

Comment:
Hi @AniBadde, I think this is the test that's getting dinged for coverage? 

```
.nox/cover/lib/python3.7/site-packages/google/auth/transport/aiohttp_req.py           76      0      8      1    99%   133->135
```

google/auth/transport/aiohttp_req.py
```py
        try:
            if self.session is None:
                self.session = aiohttp.ClientSession()
            _LOGGER.debug("Making request: %s %s", method, url)
```

It *think* you need an additional test case where no session is passed to the constructor `google.auth.transport.aiohttp_req.Request()` instead of `google.auth.transport.aiohttp_req.Request(http)`. I

Reply:
My tests currently make a sample request by defining aiohttp_req.Request(). Should I make a test where I do include the http parameter for completeness?

Reply:
^ I just did that but it still doesn't seem to see those lines in coverage, do you think we could ignore those lines in the coverage file? When I removed those lines / traced through code for the tests where the parameter isn't defined it does go through these lines which it isn't showing coverage for